lib/learn_test/client.rb

```diff
@@ -3,7 +3,7 @@
 
 module LearnTest
   class Client
-    SERVICE_URL = 'http://ironbroker-v2.flatironschool.com'.freeze
+    SERVICE_URL = ENV.fetch('IRONBROKER_URL', 'http://ironbroker-v2.flatironschool.com').freeze
```

Comment:
oh this is interesting. I was actually curious if we would be able to pass this url in as a command line argument to make it way easier to test. 

What do you think? 

This is def an improvement of the current process of opening the gem tho, so really nice work.

Reply:
That's a really cool thought. 

So I actually think I'm leaning a bit more towards an env var re testing. Given how the `learn-co` gem structure works where it's going through so many different gems, the env var will allow us to still test the implementation of `learn` to run tests rather than having to make the `learn-co` gem take the arg and pass it around. It is a bit more cumbersome to to have to do `export IRONBROKER_URL=http://qa-ironbroker`, but then we only have to do it once for the shell and all the commands should be the same.

Reply:
Ah okay, fair point. It's fine with me. Just document it.